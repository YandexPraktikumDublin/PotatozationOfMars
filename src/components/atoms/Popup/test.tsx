import React from 'react'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { Popup } from '.'

describe('<Popup />', () => {
  it('should renders correct <Popup />', () => {
    const wrapper = shallow(<Popup />)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
