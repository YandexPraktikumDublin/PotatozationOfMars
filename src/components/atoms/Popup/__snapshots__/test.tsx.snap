// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Popup /> should renders correct <Popup /> 1`] = `
<div
  className="m-auto text-center border-2 border-primary rounded-3xl p-6 bg-white bg-opacity-40 dark:text-white dark:border-white dark:bg-gray dark:bg-opacity-40"
/>
`;
